src/main/java/uk/gov/hmcts/reform/judicialapi/controller/response/UserSearchResponse.java

```diff
@@ -1,14 +1,17 @@
 package uk.gov.hmcts.reform.judicialapi.controller.response;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
-import uk.gov.hmcts.reform.judicialapi.domain.UserProfile;
+import lombok.Setter;
 
 import java.io.Serializable;
 
 @NoArgsConstructor
+@AllArgsConstructor
 @Getter
+@Setter
```

Comment:
Please check the mutation coverage for these annotations

Reply:
Ran the PiTest and didn't get any log related to the annotation in pitest report.